components/ChallengeMaterial.tsx

```diff
@@ -19,17 +23,21 @@ type ChallengeMaterialProps = {
 
 export const ChallengeTitleCard: React.FC<ChallengeTitleCardProps> = props => {
   const { currentState: state, active } = props
-  const cardStyles = []
+  const cardStyles = ['challenge-title-card']
   if (state === 'complete') {
-    cardStyles.push('challenge-title-card--done')
+    cardStyles[0] += '--done'
   } else {
     cardStyles.push('shadow-sm', 'border-0')
   }
   if (active) {
-    cardStyles.push('challenge-title-card--active')
+    cardStyles[0] += '--active'
```

Comment:
I just realized that this logic would incorrectly render a Challenge Title Card that is both Complete & Current. Maybe we don't want this to be possible, but personally I would like the idea of being able to click on my previously completed Challenges just to look at them again, thus making them look Complete & Current.

Reply:
yeah they seem like 2 different independent states, not correlated.